admin-auth/src/main/kotlin/com/epam/drill/admin/auth/config/SimpleAuthConfig.kt

```diff
@@ -0,0 +1,19 @@
+package com.epam.drill.admin.auth.config
+
+import io.ktor.config.*
+import java.lang.Boolean.parseBoolean
+
+/**
+ * A configuration for simple authentication.
+ * @param config the Ktor configuration
+ */
+class SimpleAuthConfig(
+    private val config: ApplicationConfig
+) {
+
+    /**
+     * A flag indicating whether the sign-up is enabled. Optional, true by default.
+     */
+    val signUpEnabled: Boolean
+        get() = config.propertyOrNull("signUpEnabled")?.getString()?.let { parseBoolean(it) } ?: true
```

Comment:
We have logically conflicting defaults: OAuth2 automatic sign in is enabled _and_ sign-up for simple-auth users is enabled as well.
The problem: automatic oauth2 sign-in will redirect users from UI root page right away - giving no chance to sign-up

Reply:
Set `automatic sign in` to false and add documentation (later)